Issues with building the knowledge base

Pattern-3


####################
# wso2am-publisher #
####################


###################################
# wso2am-publisher / wso2am-store #
###################################


repository/conf/axis2/axis2.xml
===============================

All hostnames are allowed. - ignore, regenerate cirts
@@ -543,7 +543,7 @@
                 <Password>wso2carbon</Password>
             </TrustStore>
         </parameter>
-        <!--<parameter name="HostnameVerifier">DefaultAndLocalhost</parameter>-->
+        <parameter name="HostnameVerifier">AllowAll</parameter>
             <!--supports Strict|AllowAll|DefaultAndLocalhost or the default if none specified -->
     </transportSender>


repository/conf/api-manager.xml
===============================

Why follow change? - ignore, configure traffic manager
@@ -484,12 +484,12 @@        
             <Password>${admin.password}</Password>
         </PolicyDeployer>
         <BlockCondition>
-            <Enabled>true</Enabled>
+            <Enabled>false</Enabled>
             <!--InitDelay>300000</InitDelay>
             <Period>3600000</Period-->
         </BlockCondition>
         <JMSConnectionDetails>
-            <Enabled>true</Enabled>
+            <Enabled>false</Enabled>
             <ServiceURL>tcp://${carbon.local.ip}:${jms.port}</ServiceURL>
             <Username>${admin.username}</Username>
             <Password>${admin.password}</Password>


repository/conf/carbon.xml
==========================

Why follow change? - ignore
@@ -582,11 +582,11 @@
     -->
     <DeploymentSynchronizer>
         <Enabled>false</Enabled>
-        <AutoCommit>false</AutoCommit>
+        <AutoCommit>true</AutoCommit>
         <AutoCheckout>true</AutoCheckout>
         <RepositoryType>svn</RepositoryType>
-        <SvnUrl>http://svnrepo.example.com/repos/</SvnUrl>
-        <SvnUser>username</SvnUser>
+        <SvnUrl>http://svnrepo/repo</SvnUrl>
+        <SvnUser>user</SvnUser>
         <SvnPassword>password</SvnPassword>
         <SvnUrlAppendTenantId>true</SvnUrlAppendTenantId>
     </DeploymentSynchronizer>


regenerate with wild cards.
repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


######################
# wso2am-key_manager #
######################


repository/conf/api-manager.xml
================================

No direct link between keymanager and gateway-manager / gateway-worker - ignore
@@ -87,13 +87,13 @@
                 <Name>Production and Sandbox</Name>
                 <Description>This is a hybrid gateway that handles both production and sandbox token traffic.</Description>
                 <!-- Server URL of the API gateway -->
-                <ServerURL>https://localhost:${mgt.transport.https.port}${carbon.context}services/</ServerURL>
+                <ServerURL>https://wso2am_gateway-manager:9443/services/</ServerURL>
                <!-- Admin username for the API gateway. -->
                 <Username>${admin.username}</Username>
                 <!-- Admin password for the API gateway.-->
                 <Password>${admin.password}</Password>
                 <!-- Endpoint URLs for the APIs hosted in this API gateway.-->
-                <GatewayEndpoint>http://${carbon.local.ip}:${http.nio.port},https://${carbon.local.ip}:${https.nio.port}</GatewayEndpoint>
+                <GatewayEndpoint>http://wso2am_gateway-worker:8280,https://wso2am_gateway-worker:8243</GatewayEndpoint>
             </Environment>
         </Environments>
     </APIGateway>

No link to analytics - ignore, * other part
@@ -136,20 +136,20 @@
 
              Ex - Multiple Receiver Groups with two receivers each
              {tcp://localhost:7612/,tcp://localhost:7613},{tcp://localhost:7712/,tcp://localhost:7713/} -->
-        <DASServerURL>{tcp://localhost:7612}</DASServerURL>
-        <!--DASAuthServerURL>{ssl://localhost:7712}</DASAuthServerURL-->
+        <DASServerURL>tcp://wso2am-analytics:7612</DASServerURL>
+        <DASAuthServerURL>ssl://wso2am-analytics:7712</DASAuthServerURL>
         <!-- Administrator username to login to the remote DAS server. -->
-        <DASUsername>${admin.username}</DASUsername>
+        <DASUsername>admin</DASUsername>
         <!-- Administrator password to login to the remote DAS server. -->
-        <DASPassword>${admin.password}</DASPassword>
+        <DASPassword>admin</DASPassword>
 
         <!-- For APIM implemented Statistic client for RDBMS -->
         <StatsProviderImpl>org.wso2.carbon.apimgt.usage.client.impl.APIUsageStatisticsRdbmsClientImpl</StatsProviderImpl>
 
         <!-- DAS REST API configuration -->
-        <DASRestApiURL>https://localhost:9444</DASRestApiURL>
-        <DASRestApiUsername>${admin.username}</DASRestApiUsername>
-        <DASRestApiPassword>${admin.password}</DASRestApiPassword>
+        <DASRestApiURL>https://wso2am-analytics:9444</DASRestApiURL>
+        <DASRestApiUsername>admin</DASRestApiUsername>
+        <DASRestApiPassword>admin</DASRestApiPassword>
 
         <!-- Below property is used to skip trying to connect to event receiver nodes when publishing events even if
             the stats enabled flag is set to true. -->

 No link to traffic-manager -  ignore
@@ -462,10 +462,10 @@
     <ThrottlingConfigurations>
         <EnableAdvanceThrottling>true</EnableAdvanceThrottling>
         <DataPublisher>
-            <Enabled>true</Enabled>
+            <Enabled>false</Enabled>
             <Type>Binary</Type>
-            <ReceiverUrlGroup>tcp://${carbon.local.ip}:${receiver.url.port}</ReceiverUrlGroup>
-            <AuthUrlGroup>ssl://${carbon.local.ip}:${auth.url.port}</AuthUrlGroup>
+            <ReceiverUrlGroup>tcp://wso2am_traffic-manager:9611</ReceiverUrlGroup>
+            <AuthUrlGroup>ssl://wso2am_traffic-manager:9711</AuthUrlGroup>
             <Username>${admin.username}</Username>
             <Password>${admin.password}</Password>
             <DataPublisherPool>
@@ -479,7 +479,7 @@
             </DataPublisherThreadPool>
         </DataPublisher>
         <PolicyDeployer>
-            <ServiceURL>https://localhost:${mgt.transport.https.port}${carbon.context}services/</ServiceURL>
+            <ServiceURL>https://wso2am_traffic-manager:9443/services/</ServiceURL>
             <Username>${admin.username}</Username>
             <Password>${admin.password}</Password>
         </PolicyDeployer>
@@ -489,8 +489,8 @@
             <Period>3600000</Period-->
         </BlockCondition>
         <JMSConnectionDetails>
-            <Enabled>true</Enabled>
-            <ServiceURL>tcp://${carbon.local.ip}:${jms.port}</ServiceURL>
+            <Enabled>false</Enabled>
+            <ServiceURL>tcp://wso2am_traffic-manager:5672</ServiceURL>
             <Username>${admin.username}</Username>
             <Password>${admin.password}</Password>
             <Destination>throttleData</Destination>

repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


##########################
# wso2am-gateway_manager #
##########################


repository/conf/axis2/axis2.xml
===============================


repository/conf/api-manager.xml
===============================

 ignore
@@ -460,7 +460,7 @@
         </WhiteListedURIs>
     </RESTAPI>
     <ThrottlingConfigurations>
-        <EnableAdvanceThrottling>true</EnableAdvanceThrottling>
+        <EnableAdvanceThrottling>false</EnableAdvanceThrottling>
         <DataPublisher>
             <Enabled>true</Enabled>
             <Type>Binary</Type>

repository/deployment/server/synapse-configs/default/api/_AuthorizeAPI_.xml - ignore
===========================================================================

@@ -5,7 +5,7 @@
        <property name="uri.var.hostname" expression="get-property('keyManager.hostname')"/>
             <send>
                 <endpoint>
-                    <http uri-template="https://{uri.var.hostname}:{uri.var.portnum}/oauth2/authorize">
+                    <http uri-template="https://wso2am_keymanager:9443/oauth2/authorize">
                         <timeout>
                             <duration>60000</duration>
                             <responseAction>fault</responseAction>

repository/deployment/server/synapse-configs/default/api/_RevokeAPI_.xml - ignore
========================================================================

@@ -5,7 +5,7 @@
        <property name="uri.var.hostname" expression="get-property('keyManager.hostname')"/>
             <send>
                 <endpoint>
-                    <http uri-template="https://{uri.var.hostname}:{uri.var.portnum}/oauth2/revoke">
+                    <http uri-template="https://wso2am_keymanager:9443/oauth2/revoke">
                         <timeout>
                             <duration>60000</duration>
                             <responseAction>fault</responseAction>

repository/deployment/server/synapse-configs/default/api/_TokenAPI_.xml - ignore
=======================================================================

@@ -5,7 +5,7 @@
        <property name="uri.var.hostname" expression="get-property('keyManager.hostname')"/>
             <send>
                 <endpoint>
-                     <http uri-template="https://{uri.var.hostname}:{uri.var.portnum}/oauth2/token">
+                     <http uri-template="https://wso2am_keymanager:9443/oauth2/token">
                         <timeout>
                             <duration>60000</duration>
                             <responseAction>fault</responseAction>

repository/deployment/server/synapse-configs/default/api/_UserInfoAPI_.xml - ignore
==========================================================================

@@ -5,7 +5,7 @@
        <property name="uri.var.hostname" expression="get-property('keyManager.hostname')"/>
             <send>
                 <endpoint>
-                    <http uri-template="https://{uri.var.hostname}:{uri.var.portnum}/oauth2/userinfo">
+                    <http uri-template="https://wso2am_keymanager:9443/oauth2/userinfo">
                         <timeout>
                             <duration>60000</duration>
                             <responseAction>fault</responseAction>

repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


#####################
# wso2am_analytics- #
#####################


repository/conf/tomcat/catalina-server.xml
==========================================


repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


#########################
# wso2am-gateway_worker #
#########################


repository/conf/axis2/axis2.xml
===============================


repository/conf/axis2/axis2_blocking_client.xml 
===============================================


repository/conf/axis2/axis2_client.xml
======================================


repository/conf/api-manager.xml
===============================

*,keymanager

 *,traffic-manager


 - self


repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


##########################
# wso2am-traffic_manager #
##########################


repository/resources/security/client-truststore.jks, repository/resources/security/wso2cabon.jks
================================================================================================
    Content has been changed but cannot read.


traffic-manager axis2.xml, registry.xml copy whole file


#########################################
# Attampting to create * Configurations #
#########################################

*,database
==========

No Issues

*,wso2am-keymanager
===================

No Issues


